mdantic.py

```diff
@@ -16,6 +16,8 @@ def __init__(self, configs=None):
             configs = {}
         self.config = {
             "init_code": ["", "python code to run when initializing"],
+            "columns": ["key, type, required, description, default",
+"Columns to use in table, comma separated"]
```

Comment:
the default value needs to be changed accordingly after previous type change

Reply:
Also the indent can be improved. Make it looks like this.
```
            "columns": [
                "key, type, required, description, default",
                "Columns to use in table, comma separated",
            ],
```
Personally I use `black` to format python code.

Reply:
Yes that's ugly, sorry about that!  I wasn't sure whether the file was black formatted already and did not want to create a huge diff. Formatted with black 23 now.  For bigger projects I use `pre-commit` so formatting is automatic for contributors and you can pin the black version in the checked in config file.